devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -0,0 +1,634 @@
+	def commands = []
+	commands << zwave.basicV1.basicGet()
+
+	if(isFlushDimmer() || isDINDimmer()) {
+		commands << zwave.meterV2.meterGet(scale: 0)
+		commands << zwave.meterV2.meterGet(scale: 2)
+	}
+	commands
+}
+
+private refreshChild() {
+	// refresh a child temperature sensor (if available)
+	if(childDevices){
+		def childDni = "${device.deviceNetworkId}:2"
+		def childDevice = childDevices.find { it.deviceNetworkId == childDni }
+
+		if (childDevice != null) {
+			sendHubCommand(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 1, scale: 0).format())
+		}
+	}
+}
+
+private encapCommands(commands, delay=200) {
+	delayBetween(commands.collect{ encap(it) }, delay)
+}
+
+private encap(cmd, endpoint = null) {
+	if (cmd) {
+		if (endpoint) {
+			cmd = zwave.multiChannelV3.multiChannelCmdEncap(destinationEndPoint: endpoint).encapsulate(cmd)
+		}
+
+		if (zwaveInfo.zw.endsWith("s")) {
+			zwave.securityV1.securityMessageEncapsulation().encapsulate(cmd).format()
+		} else {
+			cmd.format()
+		}
+	}
+}
+
+private isFlushDimmer(){
+	zwaveInfo.mfr.equals("0159") && zwaveInfo.model.equals("0051")
+}
+
+private isDINDimmer(){
+	zwaveInfo.mfr.equals("0159") && zwaveInfo.model.equals("0052")
+}
+
+private isFlushDimmer010V(){
+	zwaveInfo.mfr.equals("0159") && zwaveInfo.model.equals("0053")
+}
+
+private getParameterMap() {[
+	[
+		name: "Input 1 switch type", key: "input1SwitchType", type: "enum",
+		parameterNumber: 1, size: 1, defaultValue: 0,
+		values: [
+			0: "Default value - Mono-stable switch type (push button) – button quick press turns between previous set dimmer value and zero)",
+			1: "Bi-stable switch type ",
+			2: "Potentiometer (Flush Dimmer 0-10V is using set value the last received from potentiometer or from z-wave controller)",
+			3: "0-10V Temperature sensor (regulated output)"
+		],
+		description: "Set input based on device type (switch, potentiometer, temperature sensor,..)." +
+			"After parameter change to value 3 first exclude module (without setting parameters to default value) then wait at least 30s and then re include the module! "
+	],
+	[
+		name: "Input I1 Sensor reporting", key: "inputI1SensorReporting", type: "range",
+		parameterNumber: 140, size: 2, defaultValue: 5,
+		range: "1..10000",
+		description: "Input I1 Sensor reporting If analogue sensor is connected, module reports measured value on change defined by this parameter. " +
+			"0 Reporting disabled, " +
+			"5 (Default value) = 0,5 change, " +
+			"1 - 10000 = 0,1 - 1000 step is 0,1"
+	],
+	[
+		name: "Enable/Disable Double click function", key: "enable/DisableDoubleClickFunction", type: "boolean",
+		parameterNumber: 21, size: 1, defaultValue: 0,
+		optionInactive: 0, inactiveDescription: "Default value - Double click disabled",
+		optionActive: 1, activeDescription: "Double click enabled",
+		description: "If Double click function is enabled, a fast double click on the push button will set dimming power at maximum dimming value. " +
```

Comment:
"If enabled, a fast double-click on the push button will set the dimming level to its max."

Reply:
Fixed